tests/test_shardformer/test_model/test_shard_whisper.py

```diff
@@ -114,6 +114,7 @@ def check_forward_backward(model_fn, data_gen_fn, output_transform_fn, loss_fn,
             "tp_size": 2,
             "pp_size": 2,
             "num_microbatches": 2,
+            "enable_metadata_cache": False,
```

Comment:
This allows Whisper to send non-tensor data in pp.

Reply:
Ok, thanks!